components/ControlStation/src/de/ees/group1/cs/controller/ControlStation.java

```diff
@@ -1,30 +1,143 @@
 package de.ees.group1.cs.controller;
 
+import de.ees.group1.bt.BT_manager;
 import de.ees.group1.com.IControlStation;
+import de.ees.group1.cont.WorkStation;
 import de.ees.group1.cs.gui.IOrderController;
+import de.ees.group1.model.OrderList;
 import de.ees.group1.model.ProductionOrder;
+import de.ees.group1.model.ProductionStep;
+
 
 public class ControlStation implements IOrderController, IControlStation {
+	private ProductionOrder currentOrder;
+	private ProductionStep currentStep;
+	private int currentStepNumber;
+	private OrderList list;
+	private int statusNXT;
+	private BT_manager btManager;
+	private IControlStation cs;
+	
+	public ControlStation(){
+		btManager=new BT_manager();
+		btManager.register(cs);
```

Comment:
hä?! wo kommt denn "cs" her? INMHO wäre hier btManager.register(this) sinnvoller?!